test_scripts/API/SetGlobalProperties/026_ATF_UI_IsReady_no_resp_SetGlobalProp_UI_SetGlobalProp_UNSUP_RES.lua

```diff
@@ -0,0 +1,161 @@
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+commonSteps:PutFile("Precondition_PutFile", "action.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_SetGlobalProperties_WARNINGS_to_TTS_SGP_and_UNSUPPORTED_RESOURCE_to_UI_SGP()
+  local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+  {
+    menuTitle = "Menu Title",
+    timeoutPrompt = {{ text = "Timeout prompt", type = "TEXT" }},
+    vrHelp =
+      {{
+        position = 1,
+        image = { value = "action.png", imageType = "DYNAMIC"},
+        text = "VR help item"
+    }},
+    menuIcon = { value = "action.png", imageType = "DYNAMIC"},
+    helpPrompt = {{ text = "Help prompt", type = "TEXT" }},
+    vrHelpTitle = "VR help title",
+    keyboardProperties = { keyboardLayout = "QWERTY", language = "EN-US"}
+  })
+
+  EXPECT_HMICALL("TTS.SetGlobalProperties",
+  {
+    timeoutPrompt = {{ text = "Timeout prompt", type = "TEXT" }},
+    helpPrompt = {{ text = "Help prompt", type = "TEXT" }},
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "TTS.SetGlobalProperties", "WARNINGS", {}) end)
+
+  EXPECT_HMICALL("UI.SetGlobalProperties",
+  {
+    vrHelpTitle = "VR help title",
+    vrHelp =
+      {{
+        position = 1,
+        image = { imageType = "DYNAMIC"},
+        text = "VR help item"
+    }},
+    menuTitle = "Menu Title",
+    menuIcon = { imageType = "DYNAMIC" },
+    keyboardProperties = { keyboardLayout = "QWERTY", language = "EN-US"},
+    appID = self.applications[config.application1.registerAppInterfaceParams.appName]
+  })
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "action.png"
+    if (string.match(params.vrHelp[1].image.value, "%S*" .. "("..string.sub(storagePath, 2).."action.png)" .. "$") == nil ) then
+      print("\27[31m value of vrHelp.Image is WRONG. Expected: ~".. value_Icon .. "; Real: " .. params.vrHelp[1].image.value .. "\27[0m")
+      return false
+    else
+      return true
+    end
+  end)
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "UI.SetGlobalProperties", "UNSUPPORTED_RESOURCE", {}) end)
```

Comment:
@vvvakulenko, in requirement "[UI Interface] HMI does NOT respond to IsReady and mobile app sends RPC that must be splitted" info parameter is used. Please check it also. As example:
self.hmiConnection:SendResponse(data.id, "UI.SetGlobalProperties", "UNSUPPORTED_RESOURCE", {info = "unsupported resource"})